securedrop/models.py

```diff
@@ -124,7 +124,7 @@ def to_json(self):
             'journalist_designation': self.journalist_designation,
             'is_flagged': self.flagged,
             'is_starred': True if self.star else False,
-            'last_updated': self.last_updated.isoformat() + 'Z',
+            'last_updated': self.last_updated.isoformat() + 'Z' if self.last_updated else datetime.datetime.utcnow(),  # noqa
```

Comment:
The `Z` needs to appear outside the `if/else` otherwise it won't exist if `last_updated` is `None`.